cdc/cdc.hh

```diff
@@ -52,17 +60,48 @@ class query_state;
 
 } // namespace service
 
-namespace dht {
-
-class i_partitioner;
-
-} // namespace dht
-
 class mutation;
 class partition_key;
 
 namespace cdc {
 
+class stream_id final {
+    int64_t _token;
```

Comment:
I changed these components to `_first` and `_second`, because that's exactly what they mean. It's just a pair of ints.

In the future, one of the elements of this pair will be used to calculate the token, but that's the future, and an implementation detail anyway.